funlet-simple-message/tests/funlet-simple-message.test.js

```diff
@@ -23,76 +23,51 @@ const DEFAULT_VOICE = ALICE;
 
 const RECORDED_MESSAGE = 'https://example.com/recorded-message';
 const PLAY_RECORDED_MESSAGE =
-  XML_DECLARATION +
-  '<Response>' +
-  '<Play>' +
-  RECORDED_MESSAGE +
-  '</Play>' +
-  '</Response>';
+  `${XML_DECLARATION}<Response>` +
```

Comment:
all these multiline strings can be written to use a single starting \` and an ending `